src/Microsoft.DotNet.Darc/DarcLib/Helpers/ProcessManager.cs

```diff
@@ -76,10 +78,18 @@ public Task<ProcessExecutionResult> ExecuteGit(string repoPath, string[] argumen
             processStartInfo.ArgumentList.Add(arg);
         }
 
-        _logger.LogDebug("Executing command: '{executable} {arguments}'{workingDir}",
-            executable,
-            string.Join(' ', processStartInfo.ArgumentList),
-            workingDir is null ? string.Empty : " in " + workingDir);
+        var logMessage = $"Executing command: '{executable} {string.Join(' ', processStartInfo.ArgumentList)}'" +
+            $"{(workingDir is null ? string.Empty : " in " + workingDir)}";
+
+        if (redactedStrings != null)
+        {
+            foreach (var redacted in redactedStrings)
+            {
+                logMessage = logMessage.Replace(redacted, "[REDACTED]");
```

Comment:
what is this doing? Is it for hiding secrets in the logs?

Reply:
Yes